lib/util/source-code-obtain.js

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
The name of this file doesn't seem to match up with what's being exported. If nothing else makes sense, maybe just 'source-code-util` for now?